pkg/binfmt/Makefile

```diff
@@ -13,3 +13,11 @@ tag: $(DEPS)
 push: tag
 	DOCKER_CONTENT_TRUST=1 docker pull $(ORG)/$(IMAGE):$(HASH) || \
 	DOCKER_CONTENT_TRUST=1 docker push $(ORG)/$(IMAGE):$(HASH)
+
+check:
+	@if curl -sSL https://registry.hub.docker.com/v2/repositories/$(ORG)/$(IMAGE)/tags/$(HASH) | grep -q 'Not found'; then\
```

Comment:
for now we use hub so this has the Docker Hub notary server and registry tied to it, but this command should work for just checking one tag:
`notary -s https://notary.docker.io lookup docker.io/$(ORG)/$(IMAGE) $(HASH)`

As @justincormack points out, we can list all tags too (and grep):
`notary -s https://notary.docker.io list docker.io/$(ORG)/$(IMAGE)`